test/acceptance/behave/README.md

```diff
@@ -0,0 +1,130 @@
+     ```
+
+#### Requirements to fabric
+    ```
+     yum install gcc python-devel
+    ```
+
+#### Folders/Files Structure
+
+    components/:                         folder with several sub-folders with features and steps
+        --> common_steps/:               folder with common steps used in several features
+              -->  *.py                  files with common steps associated to previous scenarios
+        --> functionality/               sub-folder to save all features in each functionality area
+              --> environment.py:        may define code to run before and after certain events during your testing.
+              --> *.feature              files used to define scenarios
+              --> steps/                 folder of steps associated to features in the functionality
+                   -->  *.py             files with steps associated to previous scenarios
+    logs/:                               folder to log (it is created if does not exist)
+    results/:                            folder to save junit reports (it is created if does not exist)
+    settings/:                           folder to store different configurations (this place is configurable in configuration.json) (optional)
+    tools/:                              internal libraries
+    behave.ini:                          configuration files for behave are called either ”.behaverc” or “behave.ini”
+    configuration.json:                  initial configuration, before of execution
+    properties.json.base:                reference file with parameters (properties) used in tests (after is copied to properties.json)
+    README.md:                           this file, a brief explication about this framework to test
+    requirement.txt:                     external library, necessary install before to execute test (see Test execution section)
+
+### Tests execution:
+
+- Change to the test/acceptance/behave folder of the project if not already on it.
+- We recommend to create `settings` folder in  behave root directory if it does not exists and store all configurations to referenced by `properties.json` files.
+  The settings folder path could be changed in the `configuration.json`.
+  This file initially will overwrite properties.json in each feature.
+- modify in `configuration.json`:
+       * JENKINS: determine whether you are in jenkins or not.
+       * PATH_TO_SETTING_FOLDER: folder where are the different configurations
+- `properties.json` will be update automatically from settings folder (see configuration.json)
+- Run behave (see available params with the -h option).
+```
+    Some examples:
+       behave .../example.feature                      -- run only one feature
+       behave .../example.feature -t test              -- run scenarios tagged with "test" in a feature
+       behave .../example.feature -t=-test             -- run all scenarios except tagged with "skip" in a feature
```

Comment:
Both method are correct, append the info in README.md
`-t @skip`    or   `t=-skip`